pick-random-user-plugin/README.md

```diff
@@ -0,0 +1,62 @@
+# Pick random user
+
+## What is it?
+
+The Pick Random User Plugin shows a modal for moderator to pick a user (mainly viewer but it could also be a moderator) randomly out of the whole list of users present in a meetings. 
+
+![Gif of plugin demo](./public/assets/plugin.gif)
+
+## Running the Plugin from Source
+
+1. Start the development server:
+
+```bash
+cd $HOME/src/plugins/pick-random-user-plugin
+npm install
+npm start
+```
+
+2. Add reference to it on BigBlueButton's `settings.yml`:
+
+```yaml
+  plugins:
+    - name: PickRandomUserPlugin
+      url: http://127.0.0.1:4701/static/PickRandomUserPlugin.js
+      dataChannels:
+        - name: pickRandomUser
+          writePermission: ['moderator','presenter']
+          deletePermission: ['moderator', 'sender']
+        - name: modalInformationFromPresenter
+          writePermission: ['moderator','presenter']
+          deletePermission: ['moderator', 'sender']
+```
+
+## Building the Plugin
+
+To build the plugin for production use, follow these steps:
+
+```bash
+cd $HOME/src/plugins/pick-random-user-plugin
+npm install
```

Comment:
Consider `npm ci` to ensure we install the dependencies as described in `package-lock.json`

Reply:
Thank you for the suggestion, I will take a look into this!